src/Styleguide/Components/Pagination.tsx

```diff
@@ -80,7 +129,7 @@ const PageSpan = ({ mx }) => {
 const PrevButton = props => {
   return (
     <Sans size="3" weight="medium" display="inline" mx={0.5}>
-      <a href="#" className="noUnderline">
+      <a onClick={() => props.onClick()} className="noUnderline">
```

Comment:
I noticed `href='#'` was causing storybooks to register the click and navigate you in a weird way.

In the real world, this will probably update some URL (and perhaps other stuff), but that shouldn't be in this component right? That should be a callback controlled by the parent?

I wonder, for SEO purposes, we probably _do_ want to be able to render valid `href` tags as well (ones that if you load them directly, will resolve to the right page/sort/filter/etc.). That can all come later I guess?

Reply:
We'll need to think this through a bit, but ideally the links update the URL bar with a new query parameter that then updates the filter. And yup, we can do that in another pass.

Reply:
Cool, yea that can be a further pass since it for sure involves some thinking through.

Reply:
My immediate thought is that such things as interacting with the host of the app (the browser in this case) should happen at the root component level.